packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -524,6 +548,7 @@ Slider.propTypes = {
    * @ignore
    */
   theme: PropTypes.object.isRequired,
+  thumb: PropTypes.oneOfType([PropTypes.string, PropTypes.func, PropTypes.object]),
```

Comment:
Please add a block comment and document this property and use `PropTypes.element`.

Reply:
Fixed.